File: Topic Based Problems/01 Fundamentals/1 Beginner/04 Arithmetic Operations/Java/ArithmeticOperations.java
import java.util.Scanner;

public class ArithmeticOperations {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int firstNumber, secondNumber;

        System.out.print("Enter first number: ");
        firstNumber = scanner.nextInt();

        System.out.print("Enter second number: ");
        secondNumber = scanner.nextInt();

        System.out.println("Sum of " + firstNumber + " and " + secondNumber + " is: " + (firstNumber + secondNumber));
        System.out.println("Difference of " + firstNumber + " and " + secondNumber + " is: " + (firstNumber - secondNumber));
        System.out.println("Product of " + firstNumber + " and " + secondNumber + " is: " + (firstNumber * secondNumber));

        if (secondNumber != 0) {
            System.out.println("Quotient of " + firstNumber + " and " + secondNumber + " is: " + (firstNumber / (double)secondNumber));
        } else {
            System.out.println("Cannot divide by zero.");
        }

        scanner.close();
    }
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/04 Arithmetic Operations/C/ArithmeticOperations.c
#include <stdio.h>

int main() {
    int firstNumber, secondNumber;

    printf("Enter first number: ");
    scanf("%d", &firstNumber);

    printf("Enter second number: ");
    scanf("%d", &secondNumber);

    printf("Sum of %d and %d is: %d\n", firstNumber, secondNumber, firstNumber + secondNumber);
    printf("Difference of %d and %d is: %d\n", firstNumber, secondNumber, firstNumber - secondNumber);
    printf("Product of %d and %d is: %d\n", firstNumber, secondNumber, firstNumber * secondNumber);
    printf("Quotient of %d and %d is: %d\n", firstNumber, secondNumber, firstNumber / secondNumber);

    return 0;
}
